dlibs/dlib_userupdate.py

```diff
@@ -39,10 +38,53 @@ async def on_member_update(before, after, **kargs):
             pass
 
 
+async def send_joinlog(member, client, embed):
+    with db_hlapi(member.guild.id) as db:
+        if joinlog := db.grab_config("join-log"):
+            if logging_channel := client.get_channel(int(joinlog)):
+                try:
+                    logging_channel.send(embed=embed)
+                except discord.errors.Forbidden:
+                    pass
+
+
+def parsedate(indata):
+    return f"{time.strftime('%a, %d %b %Y %H:%M:%S', time.localtime(datetime.timestamp(indata)))} ({(datetime.utcnow() - indata).days} days ago)"
```

Comment:
don't you want to make a helper function or something for this?

Reply:
its wayyy too specific to make it part of lib_parsers, cause then it needs to have API guarantees, so unless its needed more or it becomes less specific it would be a bad idea to put it in parsers